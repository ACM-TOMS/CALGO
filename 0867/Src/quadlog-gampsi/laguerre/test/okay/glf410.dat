### Numerical integration with glqf()
###
### int(sin(sigma*x) * x^alpha * exp(-x) * ln(x), x = 0..infinity)
###
### Line format: 2 sigma alpha neval result relerr abserr
###
2    -1.000000000000000000E+00   1.000000000000000000E+00         50   -3.810537240921608310E-02  8.19E-13 -3.12E-14
2     1.000000000000000000E+00   0.000000000000000000E+00         50   -6.919554589202928097E-02 -1.06E-14  7.36E-16
