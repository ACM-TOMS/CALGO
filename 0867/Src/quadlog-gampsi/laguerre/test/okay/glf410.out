  Test integral: int(sin(sigma*x) * x^alpha * exp(-x) * ln(x), x = 0..infinity)

  1 ulp =   2.22E-16

  Floating-point precision appears to be (1 - log2(ulp) =) 53 bits

   nquad     sigma     alpha     Quadrature Result        Exact Integral     Rel. Error RelE (ULPs)  Err (bits)
      25   -1.0000    1.0000  -3.810537240921608E-02  -3.810537240924730E-02   8.19E-13     3689.60     11.85
      25    1.0000    0.0000  -6.919554589202928E-02  -6.919554589202855E-02  -1.06E-14      -47.87      5.58

  Maximum relative error =   8.19E-13   3.69E+03 ulps      11.85 bits at nquad =   25 sigma =   -1.0000 alpha =    1.0000
  Minimum relative error =  -1.06E-14  -4.79E+01 ulps       5.58 bits at nquad =   25 sigma =    1.0000 alpha =    0.0000
  Average relative error =   4.15E-13   1.87E+03 ulps      10.87 bits

  Done
